Footer & impressum & more

Tschonti opened the pull request:
![image](https://user-images.githubusercontent.com/13004605/219902736-41faf353-0fa6-4901-b919-4f85966e2d00.png)


![image](https://user-images.githubusercontent.com/13004605/219902737-b48fa3a9-84ee-4664-8801-5731d22d4c0c.png)

Video about the hover effects in the #arculat slack channel

Also improved the text on the impressum page, but it's not final

Closes #93 and #94

berenteb commented on src/components/commons/Footer.tsx:
```diff
@@ -1,20 +1,93 @@
-  </Box>
-)
+export const Footer: FC = () => {
+  const kirDevLogo = useColorModeValue('/img/kirdev.svg', '/img/kirdev-white.svg')
+  const hkLogo = useColorModeValue('/img/hk.svg', '/img/hk-white.svg')
```
WHy aren't you manipulating the SVG itself with inline colors or css?

Tschonti replied:
this was easier 😄

Tschonti replied:
I'm not really in the mood for editing the SVGs, so if it's okay with you, I wouldn't change this

berenteb replied:
Okay OFC

DannySS123 commented on src/components/commons/Footer.tsx:
```diff
@@ -1,20 +1,93 @@
+            </Text>
+
+            <HStack spacing={2} mt={4} justify="space-evenly">
+              <Link href="https://github.com/kir-dev" isExternal _hover={{ color: customTheme.colors.kirDev }}>
+                <FaGithub size={25} />
+              </Link>
+              <Link href="https://kir-dev.hu" isExternal _hover={{ color: customTheme.colors.kirDev }}>
+                <FaGlobe size={25} />
+              </Link>
+              <Link href="https://www.facebook.com/kirdevteam" isExternal _hover={{ color: customTheme.colors.kirDev }}>
+                <FaFacebook size={25} />
+              </Link>
+              <Link href="https://www.instagram.com/kir.dev/" isExternal _hover={{ color: customTheme.colors.kirDev }}>
+                <FaInstagram size={25} />
```
perhaps a socialLinkButton component? :D

DannySS123 commented on src/pages/impressum/ImpressumPage.tsx:
```diff
@@ -1,23 +1,46 @@
-        Aldus PageMaker including versions of Lorem Ipsum.
-      </ReactMarkdown>
+      <Text textAlign="justify">
+        Az alkalmazást a HK felkérésére Kir-Dev webfejlesztő kör készítette. Mint ahogy az összes többi projektünk, ez is nyílt forráskódú.
```
Sounds a bit strange, maybe:
Az alkalmazást a HK felkérésére a Kir-Dev webfejlesztő kör készítette. (Kir-Dev előtt "a")

DannySS123 reviewed (changes requested):
Lookin 🔥🔥🔥🔥🔥
Would have liked the color change in th logos in light mode as well, but I guess it can be counted as an easter egg in dark mode :D

And yes, for the impressum to be completed I think we'll nedd something under the images of the developers as well. But it's good for now.

Tschonti commented:
More styling changes:

- changed the avatar of consultations, now it's related to the major of the subject
![image](https://user-images.githubusercontent.com/13004605/219962674-6ea1d55b-d770-49c6-85a0-58c67675aec6.png)

- fixed scrollbar. It was previously invisble on the whole page, because it was using non-existent chakra variables (shouldn't have copied mindlessly from Trisz' repo xd). Couldn't find a way to change the color in dark mode, so the brand color doesn't look good, I think gray works fine.
![image](https://user-images.githubusercontent.com/13004605/219962742-7b45d12e-67d0-45ce-b812-c6a4ffac5b20.png)

DannySS123 reviewed (approved):
✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️✔️
